test/app-tap/datetime.test.lua

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env tarantool
+
+local tap = require('tap')
+local test = tap.test("errno")
```

Comment:
Lets be consistent and use either `''` or `""` for strings. At least on the neighbour lines.

Reply:
Corrected:

```diff
diff --git a/test/app-tap/datetime.test.lua b/test/app-tap/datetime.test.lua
index 57ca3063b..62c4eba1c 100755
--- a/test/app-tap/datetime.test.lua
+++ b/test/app-tap/datetime.test.lua
@@ -1,7 +1,7 @@
 #!/usr/bin/env tarantool
 
 local tap = require('tap')
-local test = tap.test("errno")
+local test = tap.test('errno')
 local date = require('datetime')
 
 -- local ffi = require('ffi')
@@ -244,7 +244,7 @@ test:test("totable{}", function(test)
                  tzoffset = 0, month = 1, year = 1970, hour = 0}
     local T = date.new()
     local TT = T:totable()
-    test:is_deeply(TT, exp, "date:totable()")
+    test:is_deeply(TT, exp, 'date:totable()')
 
     local D = os.date('*t')
     TT = date.new(D):totable()
@@ -252,7 +252,7 @@ test:test("totable{}", function(test)
         'sec', 'min', 'wday', 'day', 'yday', 'month', 'year', 'hour'
     }
     for _, key in pairs(keys) do
-        test:is(TT[key], D[key], ("[%s]: %s == %s"):format(key, TT[key], D[key]))
+        test:is(TT[key], D[key], ('[%s]: %s == %s'):format(key, TT[key], D[key]))
     end
     local tst_d
     for tst_d = 21,28 do
@@ -260,7 +260,7 @@ test:test("totable{}", function(test)
         D = os.date('*t', os.time{year = 2021, month = 9, day = tst_d})
         TT = date(D):totable()
         for _, key in pairs(keys) do
-            test:is(TT[key], D[key], ("[%s]: %s == %s"):format(key, TT[key], D[key]))
+            test:is(TT[key], D[key], ('[%s]: %s == %s'):format(key, TT[key], D[key]))
         end
     end
 end)

```